### PR TITLE
styles for diff on markdown preview

### LINKED ISSUE #59288: Wrong diff colors on markdown preview
- VSCode Version: 1.28.0 (1.28.0)
- OS Version: macOS High Sierra 10.13.6 (17G65)

Steps to Reproduce:

1. Create a markdown file
2. Open preview to the side

bug

Does this issue occur when all extensions are disabled?: Yes/No

Yes

### PR DESCRIPTION
**Pull request by agurod42**
Fixes #59288

### CODE REVIEW

File: src/vs/workbench/parts/webview/electron-browser/webview-pre.js [hunk 1]
```diff
@@ -258,6 +258,16 @@
 				color: var(--vscode-textPreformat-foreground);
 			}
 
+			code span.hljs-addition {
+				background: var(--vscode-diffEditor-insertedTextBackground, rgba(155, 185, 85, 0.2));
```
Comment (agurodriguez) [inline src/vs/workbench/parts/webview/electron-browser/webview-pre.js]: This default color was picked up from here: https://github.com/Microsoft/vscode/blob/d9add550e9315d133484f5497362a830a71da29a/src/vs/platform/theme/common/colorRegistry.ts#L292

File: src/vs/workbench/parts/webview/electron-browser/webview-pre.js [hunk 2]
```diff
@@ -258,6 +258,16 @@
...
+				color: rgb(155, 185, 85) !important;
+			}
+
+			code span.hljs-deletion {
+				background: var(--vscode-diffEditor-removedTextBackground, rgba(255, 0, 0, 0.2));
```
Comment (agurodriguez) [inline src/vs/workbench/parts/webview/electron-browser/webview-pre.js]: This color was picked up from here:

https://github.com/Microsoft/vscode/blob/d9add550e9315d133484f5497362a830a71da29a/src/vs/platform/theme/common/colorRegistry.ts#L293

**Review by mjbvz — CHANGES_REQUESTED** [review]
Thanks for the fix. The new css looks good but please move these into the markdown specific css file: `/extensions/markdown-language-features/media/highlight.css`

**Comment by mjbvz** [comment]
Awesome! This change will be in the next vs code insiders build and will go to stable in 1.28

**Pull request merged**